SecretMessage/deploy.yml

```diff
@@ -0,0 +1,104 @@
+---
+version: "2.0"
+
+services:
+  php:
+    image: piyushch/secretweb3:8.0.0
+    depends-on:
+      - mysql
+    env:
+      - "MYSQL_HOST=mysql"
+      - "MYSQL_USER=root"
+      - "MYSQL_PASSWORD=pass"
+      - "MYSQL_DB=moe_db"
+      - "dom=Put_your_akash_uri_here_and_update"
+    expose:
+      - port: 80
+        as: 80
+        to:
+          - global: true
+  mysql:
+    image: mysql:5.7
+    expose:
+      - port: 3306
+        as: 3306
+        to:
+          - service: php
+      - port: 3306
+        as: 3307
+        to:
+          - service: phpmyadmin
+    env:
+      - "MYSQL_ROOT_PASSWORD=rootpassword"
+      - "MYSQL_USER=root"
+      - "MYSQL_PASSWORD=pass"
+      - "MYSQL_DATABASE=moe_db"
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin:4.7
+    depends-on:
+      - mysql
+    expose:
+      - port: 80
+        as: 80
+        to:
+          - global: true
+    env:
+      - "PMA_HOST=mysql"
+      - "PMA_PORT=3307"
+
+profiles:
+  compute:
+    mysql:
+      resources:
+        cpu:
+          units: 1.0
+        memory:
+          size: 512Mi
+        storage:
+          size: 512Mi
+    php:
+      resources:
+        cpu:
+          units: 1.0
+        memory:
+          size: 512Mi
+        storage:
+          size: 512Mi
+    phpmyadmin:
+      resources:
+        cpu:
+          units: 1.0
+        memory:
+          size: 512Mi
+        storage:
+          size: 512Mi
+
+  placement:
+    westcoast:
+      signedBy:
+        anyOf:
+          - "akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63"

```

Comment:
And please add the `signedBy` attributes (two auditing keys), e.g. https://github.com/akash-network/awesome-akash/blob/b720cd5/MySQL/deploy.yaml#L29-L32 to your deployment SDL manifest.